src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -2,6 +2,14 @@ import React from 'react';
 
 import './CommentInfo.scss';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+export interface Comment {
+  name: string,
+  body: string,
+}
+
+export const CommentInfo: React.FC<Comment> = ({ name, body }) => (
```

Comment:
don't forget to fix this one
```suggestion
interface Props {
  ...
}

export const CommentInfo: FC<Props> = ({ name, body }) => (
```